ls1intum/Artemis
# Feature/user statistics

### Checklist


### Description

- Submission graph for the user statistics page
- Kept some unused server-side methods for follow-up features (see StatisticsResource, StatisticsService, StatisticsRepository)

**UPDATE**: Year, Month and Day are also implemented now

### Steps for Testing


1. Log in to Artemis
2. Navigate to user statistics in the Server Administration tab
3. Check that no errors are thrown
4. Make a submission and check that the current day in the graph has a submission more (for Week, Month), the corresponding month in the year Tab and that the time in the Day Tab is correct


### Screenshots


![Screenshot 2020-11-27 at 20 22 43](https://user-images.githubusercontent.com/44401560/100479537-cdead100-30ee-11eb-8935-1161c4708719.png)
